test_scripts/API/SetAppIcon/010_Mobile_App_received_response_SetAppIcon_REJECTED.lua

```diff
@@ -0,0 +1,66 @@
+---------------------------------------------------------------------------------------------------
+-- Proposal: https://github.com/smartdevicelink/sdl_evolution/blob/master/proposals/0041-appicon-resumption.md
+-- User story:TBD
+-- Use case:TBD
+--
+-- Requirement summary:
+-- TBD
+--
+-- Description:
+-- In case:
+-- 1) SDL, HMI are started.
+-- 2) Mobile app is registered. Sends  PutFile and valid SetAppIcon requests.
+-- 3) HMI responds with REJECTED resultCode to SetAppIcon request. Mobile App received response SetAppIcon(REJECTED).
+-- 4) App is re-registered.
+-- SDL does:
+-- 1) Register an app successfully, respond to RAI with result code "SUCCESS", "iconResumed" = true.
```

Comment:
@HSavynetska According to checklist: ```"iconResumed" = false.```

Reply:
case is updated, description is will be updated in checklist

Reply:
case is updated, description will be updated in checklist.